cmd/bblfsh/client.go

```diff
@@ -15,6 +15,11 @@ import (
 	"github.com/Sirupsen/logrus"
 	"github.com/bblfsh/sdk/protocol"
 	"google.golang.org/grpc"
+	"srcd.works/go-errors.v0"
+)
+
+var (
+	errProfiler = errors.NewKind("Failed to create a % file at %s")
```

Comment:
I agree that exposing anything in `main` doesn't seem to make sense, and I'll take a look at why is that and change it if needed, but that goes beyond this PR, so I'd expose for consistency for now.